draft-ietf-tls-tls13.md

```diff
@@ -1938,11 +1938,14 @@ processed and transmitted as specified by the current active session state.
                case certificate:           Certificate;
                case certificate_verify:    CertificateVerify;
                case finished:              Finished;
-               case session_ticket:        NewSessionTicket;
+               case new_session_ticket:    NewSessionTicket;
                case key_update:            KeyUpdate;
            } body;
        } Handshake;
 
+[[TODO: Add IANA Considerations to rename the 'NewSessionTicket' HandshakeType
+value to 'new_session_ticket'.]]
+
```

Comment:
Think of the IANA considerations as instructions for somebody changing a value in a registry table that knows nada about the protocol.

Is this what you're trying to do (add this in s11):

`IANA is requested to rename item 4 in the TLS HandshakeType Registry to 'new_session_ticket', i.e., change the table entry for value 4 to 'NewSessionTicket (renamed new_session_ticket)'.`